### PR TITLE
Add LFW format

### PR DESCRIPTION
**Pull request by kirill-sizov**
### Motivation and context

Added LFW format.

### Checklist

- [ ] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [x] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by zhiltsov-max** [comment]
Will you add the format documentation file in Datumaro?

**Review by zhiltsov-max — APPROVED** [review]

File: site/content/en/docs/manual/advanced/formats/format-lfw.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+# Import LFW annotation
+
+The uploaded annotations file should be a zip file with the following structure:
+
+```bash
+<archive_name>.zip/
+    └── annotations/
+        ├── landmarks.txt # list with landmark points for each image
+        ├── pairs.txt # list of matched and mismatched pairs of person
+        └── people.txt # list of persons name
```
Comment (nmanovic) [inline site/content/en/docs/manual/advanced/formats/format-lfw.md:28]: Isn't the file optional now?

**Comment by nmanovic** [comment]
The main question for me now: should we allow to export tasks in the format, if they don't correspond to the expected format. For example, our tasks don't have corresponding attributes. I have created a task with pets (cat, dog) without necessary attributes and exported the dataset in LFW. Did I get something useful?

**Comment by kirill-sizov** [comment]
> The main question for me now: should we allow to export tasks in the format, if they don't correspond to the expected format. For example, our tasks don't have corresponding attributes. I have created a task with pets (cat, dog) without necessary attributes and exported the dataset in LFW. Did I get something useful?

@nmanovic I think, without attributes this format can still be useful for image/face classification task.

File: site/content/en/docs/manual/advanced/formats/format-lfw.md (hunk 1 above)
Comment (kirill-sizov) [reply site/content/en/docs/manual/advanced/formats/format-lfw.md]: Yes, it's. Added clarification about it.

**Pull request merged**